packages/augur-sdk/src/Augur.ts

```diff
@@ -124,11 +129,38 @@ export class Augur<TProvider extends Provider = Provider> {
   }
 
   async getAccount(): Promise<string | null> {
-    const account = await this.dependencies.address;
+    let account;
+    if (this.dependencies.useSafe) {
+      account = this.dependencies.safeAddress;
```

Comment:
Does this need an ‘await’ to get the safeAddress?

Reply:
nope its just a property